lib/database/migrations/20240216160730-remove-obsolate-non-physics-runs-links.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const { RunDefinition } = require('../../server/services/run/getRunDefinition');
+
+const REMOVE_DATA_PASSES_LINKS_TO_NON_PHYSICS_RUNS = `
+    DELETE FROM data_passes_runs
+    WHERE run_number IN (
+        SELECT run_number
+        FROM runs as r
+        WHERE r.definition != '${RunDefinition.Physics}')
+    `;
+
+const REMOVE_SIMULATION_PASSES_LINKS_TO_NON_PHYSICS_RUNS = `
+    DELETE FROM simulation_passes_runs
+    WHERE run_number IN (
+        SELECT run_number
+        FROM runs as r
+        WHERE r.definition != '${RunDefinition.Physics}')
+    `;
```

Comment:
I would rather use an inner join than an `IN`

Reply:
applied